solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,190 @@
+        new_balance = balance_now + amount_to_deposit
+        costumers_data[costumer_id]['balance'] = new_balance
+        return True
+
+    return False
+
+
+def change_password(costumer_id, costumers_data, new_password):
+    """Changing costumer's ATM password.
+
+    Args:
+        costumer_id (string): Costumer id.
+        costumers_data (dictionary): Bank costumer data.
+        new_password (string): Password the costumer wants to change to.
+
+    Returns:
+        boolean: True for success changing the password, False otherwise.
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_data[costumer_id]['password'] = new_password
+        password_now = new_password
+        return password_now
+
+    return False
+
+
+def create_dictionary(path_of_file):
+    """Create dictionary of bank customer data.
+
+    Args:
+        path_of_file (string): Path of file that contains all bank costumer
+            data.
+
+    Returns:
+        dictionary: A nested dictionary of bank costumer data, or print a
+            message and return a None dict if the file path is not correct.
+    """
+    if os.path.isfile(path_of_file):
+        with open(path_of_file, 'r') as file_j:
+            data = json.load(file_j)
+            costumers_data = data
+            return costumers_data
+
+    print("File not found")
+    costumers_data = None
+    return costumers_data
+
+
+def check_costumer_choice(costumer_choice, costumers_data, costumer_id,
+                          costumer_password):
+    """Check costumer choice and act accordingly.
+
+    Args:
+        costumer_choice (integer): Costumer choice from 1/2/3/4/5 options,
+            references to the action he wants to do in ATM.
+        costumers_data (dictionary): Bank costumer data.
+        costumer_id (string): Costumer id.
+        costumer_password (string): Current costumer password.
+
+    Returns:
+        string: Updated password if the choice is 4, or updated id and
+            password if the choice is 5.
+    """
+    if costumer_choice == 1:
+        print(f"Balance: {costumers_data[costumer_id]['balance']}")
+
+    elif costumer_choice == 2:
+        print("Action succeeded\n" if cash_withdrawal(
+            costumer_id, costumers_data) else "Failed - check amount\n")
+
+    elif costumer_choice == 3:
+        print(
+            "Action succeeded\n" if cash_deposit(costumer_id, costumers_data)
+            else "Failed - check amount\n")
+
+    elif costumer_choice == 4:
+        new_password = input("New password: ")
+        costumer_password = new_password if change_password(
+            costumer_id, costumers_data, new_password) else costumer_password
+        return costumer_password
+
+    elif costumer_choice == 5:
+        costumer_id = input("id: ")
+        if costumer_id != '-1':
+            costumer_password = input("Password: ")
+        return costumer_id, costumer_password
```

Comment:
This function suppose to handle the users actions.
The functionality for getting a new user, isn't part of a user actions